partcad-cli/src/partcad_cli/click/commands/render.py

```
@@ -0,0 +1,137 @@
+            all_packages = ctx.get_all_packages(start_package)
+            packages = list(
+                map(
+                    lambda p: p["name"],
+                    list(all_packages),
+                )
+            )
+        else:
+            packages = [package]
+
+        for package in packages:
+            if not object is None:
```

Ruff (0.8.2)

Test for object identity should be `is not`

Convert to `is not`

(E714)